src/frontend/src/lib/api/signer.api.ts

```diff
@@ -51,3 +56,16 @@ export const signPrehash = async ({
 	const { sign_prehash } = await getSignerActor({ identity });
 	return sign_prehash(hash);
 };
+
+export const signerCanister = async ({
+	identity
+}: {
+	identity: Identity;
+}): Promise<SignerCanister> => {
```

Comment:
The api should not leak the entire canister but solely expose the function required by the consumer.

For example `getTransactions` or `getKnownUtxos` for web worker.

Reply:
My bad, seeing `export` in this place is indeed confusing - it was there as a temporary fix to satisfy the `@typescript-eslint/no-unused-vars` rule.  For clarity, I updated it to use `eslint-disable-next-line` instead. It will be of course removed later.

The plan is to add `updateBtcBalance` function that can be used by a worker in the next PR. Why not in this one - I would also need to create (or re-use the one from `address.store`) a mapper between bitcoin networks values (e.g. `mainnet` -> `{mainnet: null}`), so don't want to overload this branch.